pyls/plugins/folding.py

```diff
@@ -0,0 +1,169 @@
+# pylint: disable=len-as-condition
+
+import re
+
+import parso
+import parso.python.tree as tree_nodes
+
+from pyls import hookimpl
+
+SKIP_NODES = (tree_nodes.Module, tree_nodes.IfStmt, tree_nodes.TryStmt)
```

Comment:
I think we should also provide folding on `if` and `try/except` statements. Why did you decide not to do that?

Reply:
VSCode does individual if/else folding

Reply:
I didn't know that. Then let's follow VSCode lead in this case.